README.md

```diff
@@ -5,34 +5,35 @@
 
 ### Integration Tests
 |Branch|Debug|Release|
 |:--:|:--:|:--:|
-**master**|[![Build Status](https://ci.dot.net/buildStatus/icon?job=dotnet_roslyn/master/windows_debug_vs-integration)](https://ci.dot.net/job/dotnet_roslyn/job/master/job/windows_debug_vs-integration/)|[![Build Status](https://ci.dot.net/buildStatus/icon?job=dotnet_roslyn/master/windows_release_vs-integration)](https://ci.dot.net/job/dotnet_roslyn/job/master/job/windows_release_vs-integration/)|
-**master-vs-deps**|[![Build Status](https://ci.dot.net/buildStatus/icon?job=dotnet_roslyn/master-vs-deps/windows_debug_vs-integration)](https://ci.dot.net/job/dotnet_roslyn/job/master-vs-deps/job/windows_debug_vs-integration/)|[![Build Status](https://ci.dot.net/buildStatus/icon?job=dotnet_roslyn/master-vs-deps/windows_release_vs-integration)](https://ci.dot.net/job/dotnet_roslyn/job/master-vs-deps/job/windows_release_vs-integration/)|
-**dev16.0-preview2**|[![Build Status](https://ci.dot.net/buildStatus/icon?job=dotnet_roslyn/dev16.0-preview2/windows_debug_vs-integration)](https://ci.dot.net/job/dotnet_roslyn/job/dev16.0-preview2/job/windows_debug_vs-integration/)|[![Build Status](https://ci.dot.net/buildStatus/icon?job=dotnet_roslyn/dev16.0-preview2/windows_release_vs-integration)](https://ci.dot.net/job/dotnet_roslyn/job/dev16.0-preview2/job/windows_release_vs-integration/)|
-**dev16.0-preview2-vs-deps**|[![Build Status](https://ci.dot.net/buildStatus/icon?job=dotnet_roslyn/dev16.0-preview2-vs-deps/windows_debug_vs-integration)](https://ci.dot.net/job/dotnet_roslyn/job/dev16.0-preview2-vs-deps/job/windows_debug_vs-integration/)|[![Build Status](https://ci.dot.net/buildStatus/icon?job=dotnet_roslyn/dev16.0-preview2-vs-deps/windows_release_vs-integration)](https://ci.dot.net/job/dotnet_roslyn/job/dev16.0-preview2-vs-deps/job/windows_release_vs-integration/)|
+**master**|[![Build Status](https://dev.azure.com/dnceng/public/_apis/build/status/dotnet/roslyn/roslyn-integration-CI?label=build&branchname=master&jobname=Windows_VisualStudio_Integration_Tests&configuration=debug)](https://dev.azure.com/dnceng/public/_build/latest?definitionId=15&branchname=master&view=logs)|[![Build Status](https://dev.azure.com/dnceng/public/_apis/build/status/dotnet/roslyn/roslyn-integration-CI?label=build&branchname=master&jobname=Windows_VisualStudio_Integration_Tests&configuration=release)](https://dev.azure.com/dnceng/public/_build/latest?definitionId=15&branchname=master&view=logs)|
```

Comment:
❓ Is this link correct? When I clicked it, it went to the roslyn-CI queue.

Reply:
Good catch. That's wrong.

Reply:
@sharwell fixed

Reply:
➡️ Verified **master** links are fixed.